### ISSUE TITLE
How do I get non-US bank support working?

### ISSUE THREAD

**Issue by logan-life** [kevinschaich/mintable#165]
I see the instructions in the README which say:

> * Fork & edit the [country codes here](https://github.com/kevinschaich/mintable/blob/377257a6040ed9b6dd93d88435e53c48108b5806/src/integrations/plaid/plaidIntegration.ts#L126). Default support is for US banks.

I forked the repo and edited the file to "NL" which is the correct country code for the Netherlands.

That's kind of as far as I've gotten, though. How do I actually run the forked repo with this change and get things setup?

**Comment by logan-life** [comment]
Okay, I (partially) figured out how to make progress.

After forking and cloning, install mintable globally

`npm install -g mintable`

then follow the instructions in the development section to link the global install to the local copy:

```
npm install
npm run build
npm link
```

After doing this, go through the account-setup process and the NL banks showed up.

Unfortunately, I could not successfully link either of my banks. I tried both ABN AMRO and bunq. In both cases, I hit this error screen:

![image](https://github.com/kevinschaich/mintable/assets/24423298/6e3fecd3-a83e-4e6f-b606-ae55974bb410)

and get this console output:
```
2023-09-01T14:08:15.863Z [INFO] Successfully opened configuration file.
2023-09-01T14:08:15.863Z [INFO] Successfully parsed configuration.
2023-09-01T14:08:16.864Z [INFO] Successfully validated configuration.
2023-09-01T14:08:16.865Z [INFO] Account setup in progress.
2023-09-01T14:08:23.556Z [INFO] Successfully created Plaid link token.
```

The app then just hangs and no more progress can be made. Any advice welcome.

**Comment by logan-life** [comment]
I'm able to run `mintable fetch` and everything works, it just can't pull any transaction data because I can't get the account linkage working.

**Comment by logan-life** [comment]
I was able to get more detailed error information on this by viewing the error logs inside my plaid account.

The error I am getting is the [Addition Limit](https://plaid.com/docs/errors/rate-limit-exceeded/#addition_limit) which suggests to me that I need to get manually verified by Plaid first.

I didn't know I needed to do that. Clicking on the Development Environment in my Plaid dashboard popped up a thing requesting my phone number and a description of what I'm building. I filled that in and am awaiting the approval. This might unblock me!
